radar/2025-11-21/cht-studio.md

```diff
@@ -0,0 +1,10 @@
+---
+title:      "CHT Studio"
+ring:       trial
+quadrant:   languages-and-frameworks
+tags:       [app-building]
+---
+
+[CHT Studio](https://github.com/Astrozyt/cht-studio) is a graphical interface to configure and preview CHT projects without requiring a running CHT server.
```

Comment:
I think that if we are going to include this we should probably include a disclaimer that it is still in heavy development and not ready for production usage.